.circleci/config.yml

```diff
@@ -2,65 +2,271 @@ version: 2.1
 orbs:
   cypress: cypress-io/cypress@1
 jobs:
-  build-and-test:
+  preflight:
+    docker:
+      - image: circleci/python:3.7.3-node-browsers
+        environment:
+          DATABASE_URL: postgresql://root@localhost:5432/arlo-test
+          FLASK_ENV: test
+      - image: circleci/postgres:9.6.3-alpine-ram
+        environment:
+          POSTGRES_USER: root
+          POSTGRES_DB: arlo-test
+    resource_class: xlarge
```

Comment:
how did you choose resource_class? do all the jobs benefit from this class? also do they cost diff amounts of money or is it all the same?

Reply:
from what I can tell the larger the resource class the faster the tests run. We can probably knock down some of the already fast steps down. Going from medium -> xlarge saves us about a minute of compute time. And it uses more credits, but I don't think we're coming close to our alotted limit that would require us to pay more (except when I'm testing these configs :sweat_smile:)

Reply:
cool, sounds fine then